src/components/ui/Input.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+const Input = (props) => {
+    const {
+        handleChange,
+        value,
+        id,
+        name,
+        type,
+        isRequired = false,
+        placeholder,
+        label,
+        position = "top",
+        labelFontSize = "5xl",
+        labelFontWeight = "normal",
+        radius = "md",
+        inputWidthSize = "",
+        inputHightSize = "",
+        inputFontSize = "3xl",
+        inputFontWeight = "light",
+
+        ...rest
+    } = props;
+
+    const inputContainerClasses =
+        position === "left" || position === "right"
+            ? "flex items-center "
+            : "block ";
+
+    const inputClasses = `border border-solid border-gray-300 px-[20px] py-[20px] rounded-${radius} 
+    ${inputWidthSize ? `w-${inputWidthSize} ` : ""} ${
+        inputHightSize ? `h-${inputHightSize} ` : ""
+    } 
```

Comment:
1. Since in above props we give `inputWidthSize` a default value, so this condition for (inputWidthSize) could be better if removed.
2. (inputHightSize) could be better if removed, I know that in figma designs it differs from one input to another, but it could be better to be standarized.